lib/Payment.js

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+/**
+ * Payment is class that tracks payments from various providers.
+ * It can be used on the front and back end to ensure consistency.
+ * If you post to a bedrock-payment-http route and don't use this class
+ * it could be rejected as invalid.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.id - An id to use for the payment.
+ * @param {string} options.amount - The amount of the payment.
+ * @param {string} options.creator - The payee's account id.
+ * @param {string} options.service - The payment service to use.
+ * @param {string} [options.currency = 'USD'] - The currency of the payment.
+ * @param {Array<string|object>} options.orders - The productIds of the products
+ *   being purchased.
+ * @param {string} [options.stats = PaymentStatus.PENDING] - The payment status.
```

Comment:
does `PaymentStatus.PENDING` correspond to something? I see the status in the class is set to the string 'PENDING'.

Reply:
It does correspond to a set of statuses on the back end: https://github.com/digitalbazaar/bedrock-payment/blob/initial/lib/constants.js

I did change the comment to use 'PENDING' as the default value, but I actually think in this case referencing PaymentStatus might be more information for a developer in the future.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-web-payment/pull/1/commits/0618c257af143d5a0adf0ea68f7f4f84c59f9358